docs/stacks-guide-rook.md

```diff
@@ -0,0 +1,874 @@
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+EOF
+
+kubectl apply -f rook-operator.yaml
+```
+
+You can view the status of the `KubernetesApplication` as its resources are
+created:
+
+*Command*
+```bash
+kubectl get kubernetesapplication -n yugastore-app
+```
+*Output*
+```bash
+NAME            CLUSTER         STATUS      DESIRED   SUBMITTED
+rook-yugabyte   yugastore-k8s   Submitted   7         7
+```
+
+You can also view the individual `KubernetesApplicationResources` as they are
+created:
+
+*Command*
+```bash
+kubectl get kubernetesapplicationresources -n yugastore-app
+```
+*Output*
+```bash
+rook-app-namespace    Namespace                  rook-yugabytedb                 yugastore-k8s   Submitted
+rook-clusterrole      ClusterRole                rook-yugabytedb-operator        yugastore-k8s   Submitted
+rook-crds             CustomResourceDefinition   ybclusters.yugabytedb.rook.io   yugastore-k8s   Submitted
+rook-namespace        Namespace                  rook-yugabytedb-system          yugastore-k8s   Submitted
+rook-serviceaccount   Deployment                 rook-yugabytedb-operator        yugastore-k8s   Submitted
+```
+
+It is not unusual for the `rook-yugabytedb-operator` to fail on first attempt.
+It is reliant on the `ClusterRole` and will fail if it is not present when the
+container starts. However, the Crossplane `workload` controllers will take care
+of recreating it for us!
+
+## Deploy Yugastore into Target Cluster
+
+### Provision YugabyteDB with a Resource Claim
+
+Now that we have a GKE cluster up and running with the Rook Yugabyte operator installed, we can create our YugabyteDB
+cluster.
+
+```bash
+cat > yugabyteclaim.yaml <<EOF
+apiVersion: database.crossplane.io/v1alpha1
+kind: PostgreSQLInstance
+metadata:
+  name: yugastore-db
+  namespace: yugastore-app
+  labels:
+    app: yugastore
+spec:
+  classSelector:
+    matchLabels:
+      app: yugastore
+  writeConnectionSecretToRef:
+    name: yugastore-db-secret
+EOF
+
+kubectl apply -f yugabyteclaim.yaml
+```
+
+You can view the status of the `PostgreSQLInstance` claim as it waits for the
+`YugastoreCluster` to come available:
+
+*Command*
+```bash
+kubectl get postgresqlinstances -n yugastore-app
+```
+*Output*
+```bash
+NAME           STATUS   CLASS-KIND             CLASS-NAME     RESOURCE-KIND     RESOURCE-NAME                      AGE
+yugastore-db   Bound    YugabyteClusterClass   yuga-cluster   YugabyteCluster   yugastore-app-yugastore-db-t9xmf   8m38s
+```
+
+You can view the status of `YugabyteCluster` itself as it is created:
+
+*Command*
+```bash
+kubectl get yugabyteclusters.database.rook.crossplane.io yugastore-app-yugastore-db-t9xmf
+```
+*Output*
+```bash
+NAME                               AGE
+yugastore-app-yugastore-db-t9xmf   12m
+```
+
+### Provision Yugastore with a `KubernetesApplication`
```

Comment:
comment from above, suggest using just plain text for KubernetesApplication without the inline code markdown for the heading